### ISSUE TITLE
implement tab_line theme property in preview

### ISSUE THREAD

**Issue by Dani3lNL** [mozilla/addons#803]
Currently the preview is a very very basic color preview, not giving an impression of what it actually looks like. (example, [in this one](https://addons.mozilla.org/en-US/firefox/addon/dark-blue-f-theme/) it's just blue and white, instead of showing the actual URL bar being blue, current-tab line being light blue, ...)

**Comment by jvillalobos** [comment]
We don't plan to allow devs to upload preview images because it would lead to all kinds of issues. However, the preview image should reflect what the theme does.

This is what I see when that theme is applied (on Nightly):

<img width="1148" alt="screen shot 2018-10-03 at 11 33 12 am" src="https://user-images.githubusercontent.com/62568/46428096-54b6e580-c700-11e8-99f7-c5ff79484f67.png">

The only difference I see is the current-tab line, which should be blue and not grey.

**Comment by Dani3lNL** [comment]
Yes, it's correct showing up on your Nightly. My point: when a user browses through themes on addons.mozilla.org he always needs an extra click on "install theme" to see what it really will look like, which makes it harder for me as a creator to convince users to use my theme/gain more theme-users.

**Comment by Dani3lNL** [comment]
I don't understand Mozilla Addons Team's policy about difference between "extensions" and "webExtensions (themes)". But hope in future that we can implement better ways :)

**Comment by eviljeff** [comment]
@jvillalobos is the request here just to add the line to the top of the current tab? (`tab_line` I think)

**Comment by jvillalobos** [comment]
The request is to support that property in the preview, not necessarily in the wizard. One thing that confuses me about the wizard now is that changing "Header area text and icons" also changes that line in the preview. Is that a bug?

**Comment by jvillalobos** [comment]
The docs don't say anything about it, and as far as the last ESR that doesn't appear to be the behavior. So, I think that we need to fix two things: `textcolor` shouldn't change the tab line, and `tab_line` should.

**Comment by AlexandraMoga** [comment]
Verified fixed on AMO -dev with FF62, ESR 60 and Nightly 64

1. Wizard 
- `tab_line` is grayed out by default and the `textcolor` value will have no effect on it

![image](https://user-images.githubusercontent.com/31961530/46730365-990e2c80-cc90-11e8-9fdc-25ff021474cb.png)

![image](https://user-images.githubusercontent.com/31961530/46731153-a3c9c100-cc92-11e8-9f99-90824cf86c64.png)

2. Package submission

![image](https://user-images.githubusercontent.com/31961530/46731593-c4dee180-cc93-11e8-8c05-f70575ca9cc8.png)

@eviljeff I've also observed something regarding the `textcolor` field in the wizard:

- in the generated preview, the selected `textcolor` is applied to text displayed in the header area
- after installation, the selected `textcolor` is also applied to toolbar area text and icons

![image](https://user-images.githubusercontent.com/31961530/46732070-23588f80-cc95-11e8-94c0-4032c0dc422d.png)

This is not a regression from your changes, I just didn't notice this before. 
It could be an issue with the way Firefox is rendering the preview though.
